code/modules/plumbing/ducts.dm

```diff
@@ -194,6 +194,7 @@ All the important duct code:
 /obj/machinery/duct/proc/lose_neighbours()
 	for(var/obj/machinery/duct/D in neighbours)
 		D.neighbours.Remove(src)
+		D.generate_connects()
 	neighbours = list()
```

Comment:
While you're here, can you come up for a better var name than `D`? My lil' rule is that if you touch a shitty var name, you should fix it while you're there.

Reply:
> Very sorry, but... that is also a var name I have to pester about since it is in PascalCase and our vars should stick to snake_case

can you give me an example please?